src/components/table/helpers/mixin-tbody-row.js

```diff
@@ -276,9 +297,32 @@ export default {
         }
 
         // Render the details slot in a TD
-        const $details = h(BTd, { props: { colspan: fields.length }, class: this.detailsTdClass }, [
-          this.normalizeSlot(detailsSlotName, detailsScope)
-        ])
+        var $details = !this.computedFields.some(e => e.fieldDetails)
```

Comment:
Perhaps `this.computedFields.some(e => e.fieldDetails)` should be a computed prop, so that it isn't evaluated on every row render, for performance reasons.

Reply:
Got it. Fine.
.. I didn't noticed I could do this in this context 

I did the same for 
```
this.computedFields.reduce(
          (res, e) => (e.fieldDetails && [...res, `row-details(${e.key})`]) || res,
          []
        )
```

Is this ok ?